This patch fixes a double-free that the 1.2.12.06 memleak patch
accidently introduced.  This is a non-critical double-free; it will
actually never be triggered because the underlying js_alloc() (called by
js_create()) will terminate MaraDNS should she be unable to 
allocate memory.  

--- maradns-1.3.06/server/MaraDNS.c.orig	2007-06-10 10:34:42.000000000 -0500
+++ maradns-1.3.06/server/MaraDNS.c	2007-06-10 10:35:13.000000000 -0500
@@ -3122,7 +3122,6 @@
         return JS_ERROR;
         }
     if((origq = js_create(256,1)) == 0) {
-        js_destroy(lookfor);
         udperror(sock,raw,0,0,SERVER_FAIL,"can't create origq string",2,
 			desires_recursion,ect,1);
         js_destroy(lookfor);
